spec/rails_helper.rb

```diff
@@ -7,6 +7,7 @@
 abort('The Rails environment is running in production mode!') if Rails.env.production?
 require 'rspec/rails'
 require 'webmock/rspec'
+WebMock.disable_net_connect!
```

Comment:
Can you explain this as well?

Reply:
Yeah, during the testing we do not want to contact the actual internet, so we want to disable the connection. However we can during manual testing enable this to be able to see our requests are working correctly.

Reply:
i see 👍